src/Components/Patient/PatientRegister.tsx

```diff
@@ -1057,6 +1057,9 @@ export const PatientRegister = (props: PatientRegisterProps) => {
                       value={state.form.name}
                       onChange={handleChange}
                       errors={state.errors.name}
+                      inputProps={{
+                        autoComplete: "new-password",
+                      }}
```

Comment:
🤔 How is this fixing the issue?

Reply:
Found the solution for this issue here: https://material-ui.com/components/autocomplete/